packages/dso-toolkit/src/components/card/card.models.ts

```diff
@@ -3,12 +3,15 @@ import { Label } from "../label/label.models.js";
 import { Selectable } from "../selectable/selectable.models.js";
 import { Toggletip } from "../toggletip/toggletip.models.js";
 
+export type ImageShape = "square" | "rectangle";
```

Comment:
Dit lijkt niet in gebruik maar is misschien wel een heel goed idee om te doen ipv wideImage. Dan kunnen toekomstige afmetingen niet-breaking toegevoegd worden.

```
export type ImageShape = "normal" | "wide";
```


```
imageShape: ImageShape = "normal"
```